content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -300,6 +383,27 @@ token:          ...
 If you launch a new Pod into the `examplens` namespace, it can use the `myserviceaccount`
 service-account-token Secret that you just created.
 
+{{< note >}}
+The way that Kubernetes is able to tell the difference between a manually
+created legacy token and an auto-generated legacy token is by assuming that
+only auto-generated legacy tokens are bi-directionally referenced by the
+ServiceAccount. For example:
```

Comment:
```suggestion
Kubernetes distinguishes auto-generated legacy tokens from manually created legacy tokens by
checking for a reference from the ServiceAccount's `secrets` field. For example:
```

Reply:
According to @zshihang's suggestion, I added one paragraph to explain the difference of auto-generated tokens and the manually generated tokens in the `Auto-generated legacy ServiceAccount token clean up` section. Thus I deleted the difference here. I also revised the "note" to the "warning" to acquire the user's attention. (I added the warning here as previously I saw the stack-overflow discussion about manually creating the legacy token. They think it's necessary to add the secret name to the service account 'secerts' field after creating the secret.)

Reply:
Thanks, but please change the `warning` back to a `note`. Maybe a `caution`.

`warning` is reserved for “you must read this specific detail or your cluster will not be secure” levels of attention, eg “**Warning**: _do not copy the private key for the trust anchor and commit it to a public Git repo_”.

Reply:
Revised to 'caution'.